examples/libs/rlottie/lv_example_rlottie_1.c

```diff
@@ -8,7 +8,9 @@
 void lv_example_rlottie_1(void)
 {
     extern const uint8_t lv_example_rlottie_approve[];
-    lv_obj_t * lottie = lv_rlottie_create_from_raw(lv_scr_act(), 100, 100, (const void *)lv_example_rlottie_approve);
+    extern const size_t lv_example_rlottie_approve_size;
+    lv_obj_t * lottie = lv_img_create(lv_scr_act());
+    lv_rlottie_from_raw(lottie, 100, 100, (const void *)lv_example_rlottie_approve, lv_example_rlottie_approve_size, 0);
```

Comment:
Absolutely, and it works also. It's not shorter however, since you must call `lv_obj_set_size` before `lv_img_set_src`.

This example is to show the specific parameters (size & rendering buffer size)

Reply:
Great, thanks. 

I think it'd be useful to demonstrate or at least document in a comment that `lv_img_set_src` also works.

Reply:
Ok